angular/src/lib/radio/examples/examples.module.ts

```diff
@@ -1,27 +1,23 @@
-import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-
 import { RadioModule } from '@momentum-ui/angular';
 import {
   ExampleRadioDefaultComponent,
   ExampleRadioDisabledComponent,
   ExampleRadioNestedComponent,
 } from './index';
-import { RadioGroupComponent } from 'src/lib/radio-group';
 
```

Comment:
One of the radio examples uses the RadioGroupComponent

Reply:
Was just an examples module issue in importing the module to provide the component - updated.